cli/command/image/build_linux_test.go

```diff
@@ -0,0 +1,54 @@
+//+build linux
+
+package image
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"syscall"
+	"testing"
+
+	"github.com/docker/cli/internal/test"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/gotestyourself/gotestyourself/fs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
+	dest := fs.NewDir(t, "test-build-context-dest")
```

Comment:
Should'nt we run this test on Darwin as well ?

Reply:
@simonferquel yes, this is also on my TODO (but I should open an issue to track the work on that)